.github/workflows/workflow.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+FILE_PATH="$1"
+
+# Arguments other than file path are passed to the command's arguments
+shift 1
+
+if [ ! -f "$FILE_PATH" ]; then
+    echo "No such file: $FILE_PATH"
+    exit 1
+fi
+
+exit_status=0
+
+while IFS= read -r line
+do
+    ros2 launch scenario_test_runner scenario_test_runner.launch.py scenario:="$line" "$@"
+    ros2 run scenario_test_runner result_checker.py /tmp/scenario_test_runner/result.junit.xml
+    cmd_exit_status=$?
```

Comment:
```suggestion
    command_exit_status=$?
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/1223/commits/de2d26ed92f7df11a520b35a01949fcc7de31426